justfile

```diff
@@ -2,7 +2,8 @@ migrate:
     uv run manage.py makemigrations && uv run manage.py migrate
 
 run:
-    uv run manage.py runserver 0.0.0.0:8888
+    uv run manage.py collectstatic --noinput
+    DEBUG=True DJANGO_SECRET_KEY=secret uv run manage.py runserver 0.0.0.0:8888
```

Comment:
@maddyblue What are your feelings about a template .env file? The just recipe could automatically copy it if .env isn't provided by the user.

I'm using a .env to provide, e.g., social login credentials for local dev.

Reply:
SGTM! Added an env file to the dev container.